instances/widgets.treasury-factory.near/widget/pages/lockup/SettingsDropdown.jsx

```diff
@@ -0,0 +1,115 @@
+const { selectedValue, onChange, disabled, isPendingPage } = props;
+
+onChange = onChange || (() => {});
+
+const columnsVisibility = JSON.parse(
+  Storage.get(
+    "COLUMNS_VISIBILITY",
+    `${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/pages.lockup.SettingsDropdown`
+  ) ?? "[]"
+);
+
+function changeColumnsVisibility(value) {
+  Storage.set("COLUMNS_VISIBILITY", JSON.stringify(value));
+}
+
+const baseColumns = [
+  { title: "Created At", show: true },
+  { title: "Recipient Account", show: true },
+  { title: "Token", show: true },
+  { title: "Amount", show: true },
+  { title: "Start Date", show: true },
+  { title: "End Date", show: true },
+  { title: "Cliff Date", show: true },
+  { title: "Allow Cancellation", show: true },
+  { title: "Allow Staking", show: true },
+  { title: "Required Votes", show: true },
+  { title: "Votes", show: true },
```

Comment:
it's not visible in 'History' tab, so you need to hide it if settings dropdown is rendered inside history tab